test/utils/mock-lcms-get-airtable.js

```diff
@@ -945,6 +945,17 @@ function mockLcmsAirtableData() {
         content: { dareChallenges: [], steps: [] },
       },
     ],
+    'translations': [
+      {
+        id: 1,
```

Comment:
```suggestion
        id: 'challenge.id.solutionToDisplay',
```
Quitte à être cohérent avec la réalité…